buddyboss/buddyboss-platform
# Group Description html encoding

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2615  .

### How to test the changes in this Pull Request:

Steps to reproduce the behavior:
1. Go to Group Manage Screen
2. Update Group Desc with HTML content
3. Go to activity feeds screen
4. See error

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
